docs/CHANGELOG.asciidoc

```diff
@@ -71,6 +71,23 @@ To display the cluster data in *Discover*, load documents directly from `_source
 . Go to `discover:searchFieldsFromSource`, then select *On*.
 ====
 
+[discrete]
+[[breaking-kibana-discover-fields]]
+.*Discover* now uses the fields API
+[%collapsible]
+====
+*Details* +
+In 7.12.0, *Discover* uses the {ref}/search-fields.html#search-fields-param[fields API] by default.
+Reading from `_source` is still supported through a switch in *Advanced Settings*.
+This change stems from updates made to {es} in 7.11 with the extension of
+the search API to include the new {ref}/search-fields.html#search-fields-param[fields] parameter.
```

Comment:
This sentence seems to be repeating information, maybe we can delete it and add a one-sentence summary of the change:

```suggestion
With the fields API, Discover now shows all fields including runtime fields, multi-mapped fields like `.keyword`, and alias fields.
```